documents-webapp/src/main/webapp/vue-app/documents/components/body/table/cells/DocumentsFileNameCell.vue

```diff
@@ -74,7 +74,7 @@ export default {
           class: 'fas fa-file-image',
           color: '#999999',
         };
-      } else if (type.includes('video') || type.includes('octet-stream') || type.includes('ogg')) {
+      } else if (type.includes('video') || type.includes('vob') || type.includes('ogg') || type.includes('m2v') || type.includes('mjpeg') || type.includes('asf')) {
```

Comment:
Why not define an array of types, example: viedoTypes: [vob, ogg, m2v...], another one for images imageTypes: [], documentTypes..., and fix test to: viedoTypes.includes(type)